plugin.video.srfplaytv/resources/language/German/strings.po

```diff
@@ -0,0 +1,128 @@
+# Kodi Media Center language file
```

Comment:
same: `resources/language/resource.language.de_de/strings.po`

Reply:
Same question as above. Should I put this into the folder `resources/language/resource.language.de_ch`?

Reply:
Same thing :) give it a go